src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IncentiveReview.kt

```diff
@@ -24,7 +24,7 @@ data class IncentiveReview(
   val negativeBehaviours: Int,
 
   @Schema(description = "Whether the prisoner has an ACCT open alert", example = "true")
-  val acctOpenStatus: Boolean,
+  val acctStatus: Boolean,
```

Comment:
What was the rational for this rename? "acctOpenStatus = true" kinda reads better (because it's open or closed) than "acctStatus = true".

Maybe being on an ACCT status is how they call having an ACCT alert?

Reply:
no good reason 😉. it's what was in the UI app contract. not sure either are the "best". maybe `hasAcctOpen`????